### ISSUE TITLE
UserColumn.isNamed always returns true

### ISSUE THREAD

**Issue by fjlopez** [ngageoint/geopackage-core-java#39]
This code has a bug that makes it to return `true`:
https://github.com/ngageoint/geopackage-core-java/blob/270a321e6982fd5d08b0093e5e4e44200a82520f/src/main/java/mil/nga/geopackage/user/UserColumn.java#L105-L107

Parameter `name` hides member field `name`.  Additionally, we should avoid potential NPE. It should be:
```java
 public boolean isNamed(String name) {
    return this.name != null && this.name.equals(name);
 } 
```

**Comment by bosborn** [comment]
Nice find, thanks.

A database column name should not be null, so that check seems unnecessary.

**Comment by fjlopez** [comment]
I could be a good idea to make it explicit via a `@NotNull` annotation in the constructor

**Comment by bosborn** [comment]
Fixed in [3.1.0](https://github.com/ngageoint/geopackage-core-java/releases/tag/3.1.0)
